webapp/channels/src/components/drafts/drafts.scss

```diff
@@ -13,7 +13,6 @@
     }
 
     &__main {
-        position: absolute;
```

Comment:
this style was breaking the ui in case of team edition version when only drafts was being shown.